apis/policies/v1/policyserver_types.go

```diff
@@ -65,6 +75,13 @@ type PolicyServerSpec struct {
 	// key named verification-config in the Configmap.
 	// +optional
 	VerificationConfig string `json:"verificationConfig,omitempty"`
+
+	// Security configuration to be used in the Policy Server workload.
+	// The field allows different configurations for the pod and containers.
+	// This configuration will not be used in containers added by other
+	// controllers (e.g. telemetry sidecars)
+	// +optional
+	Security PolicyServerSecurity `json:"security,omitempty"`
```

Comment:
Nitpicking, and annoying maybe this late in the process:

```suggestion
	SecurityContexts PolicyServerSecurity `json:"securityContexts,omitempty"`
```

This way looking at `PolicyServer.spec.securityContexts.container` & `PolicyServer.spec.securityContexts.pod` is self-explainable.